packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleTextViewer/SingleTextViewer.stories.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { storiesOf } from '@storybook/react'
```

Comment:
I missed this the first time through. `storiesOf` isn't supported by storybook any more, except for legacy code. New stories should be using Component Story Format.